src/System.Management.Automation/help/HelpCommands.cs

```diff
@@ -647,6 +666,43 @@ private void LaunchOnlineHelp(Uri uriToLaunch)
             }
         }
 
+#if !UNIX
+        private bool WebBrowserAvailable()
+        {
+            RegistryKey regKey = null;
+            try
+            {
+                // Query the registry and remove the quotes '"' from the path.
+                regKey = Registry.ClassesRoot.OpenSubKey("HTTP\\shell\\open\\command", false);
```

Comment:
there are a few problems with this method:
1. `WebBrowserAvailable` should be renamed to `IsWebBrowserAvailable`. If you want to use the default browser executable path, then it should be renamed to `GetDefaultWebBrowser`
2. `regKey` could be `null`, on Nano or IoT. So your current code might raise `NullReferenceException`
3. The registry path `HKCR\\HTTP\\shell\\open\\command` is not pointing to the default browser. It's actually always pointing to IE. The default web browser executable path can be found with 2 steps:
      (1) HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\Shell\Associations\UrlAssociations\http\UserChoice -- get the value for `ProgId` of this key, say the value is `ChromeHTML`
      (2) HKEY_CLASSES_ROOT\ChromeHTML\shell\open\command -- get the value of the default name. The value would be the path to chrome.exe
If we want to run the web browser executable directly, then we need to make sure we really get the default browser.